esmvalcore/cmor/_fixes/native6/era5.py

```diff
@@ -204,6 +228,30 @@ def fix_metadata(self, cubes):
         return cubes
 
 
+class Rsns(Fix):
+    """Fixes for Rsns."""
+    def fix_metadata(self, cubes):
+        """Fix metadata."""
+        for cube in cubes:
+            fix_hourly_time_coordinate(cube)
+            fix_accumulated_units(cube)
+            cube.attributes['positive'] = 'down'
+
+        return cubes
+
+
+class Rsus(Fix):
+    """Fixes for Rsus."""
+    def fix_metadata(self, cubes):
+        """Fix metadata."""
+        for cube in cubes:
+            fix_hourly_time_coordinate(cube)
+            fix_accumulated_units(cube)
+            cube.attributes['positive'] = 'up'
```

Comment:
I think it's best we change these (the positive attribute) directly in the custom CMOR tables, could you do that please? Cheers for cleaning up my old mistakes :beer:

Reply:
I'm not sure what you mean by that (sorry I'm a bit out of my depth here - first time working with the core). Setting the attribute here is not necessary, is that what you are saying? I just copied the exact syntax from the other fixes. So do you want me to remove all of the lines where attributes are set?

Reply:
That's done in #1051

Reply:
@lukasbrunner sorry for being a bit too technical, mate - yeah, the changes I was suggesting were to be done straight in the custom CMOR tables (we have CMOR tables that are standard for the standard variables, and those are the PCMDI CMOR tables, and we have custom ones that we construct to include all these types of non-standard variables, and the ones for the rsus etc variables were written by me when I was young and innocent and made those attribute mistakes) :+1:

Reply:
Right thanks for explaining @valeriupredoi  :) I may have another one for you: #1054